### DISCUSSION TITLE
Értesítés beküldött bevallásról

### DISCUSSION THREAD

**Discussion by kdenes6500** [nav-gov-hu/M2M#29] [Kérdések] [answered]
Az M2M rendszeren keresztül beküldött bevallásról, bejelentésről (pl. 24T1042E-ről) nem érkezik semmilyen üzenet a tárhelyre (feldolgozási, letöltési igazolás vagy elfogadó/elutasító nyugta).
Pl. tegnap tesztelésként 5 db 24T1042E-t küldtem be azonos adattartalommal. Az ÜPO-n az érintett cégnél a Foglalkoztatással kapcsolatos bejelentések - Foglalkoztatóként tett bejelentések - Egyszerűsített foglalkoztatás menüpontban mind az 5 db látszódik (az első feldolgozott, a 2-5. pedig hibás státuszban).

Ez csak egy átmeneti hiba vagy nem is várható a megszokott módon értesítés (nyugta) a tárhelyre az M2M-en küldött fájlokról?

**Comment by cogentax** [comment]
Mi is most küldtünk be nyomtatványokat az éles rendszerben, és pontos ezt a kérdést szerettem volna feltenni.  Vszínűleg minden ok volt, mert viszakaptuk a File_id-t és az Arrival number-t, de mi sem kaptunk tárhely üzenetet.
Érkezik-e visszajelzés a tárhelyre vagy nem?

**Comment by TamasPill** [comment]
Igen, ez egy probléma most, hogy nem kerülnek kiküldésre a befogadó nyugták. 
Ha minden jól megy, akkor ez holnap már javítva is lesz.

**Comment by TamasPill** [comment, marked as answer]
Tegnap este települt javítás a fenti problémára.
Most már a beküldő megkapja a tárhelyére a befogadó/elutasító nyugtát.

**Comment by cogentax** [comment]
Teszteltük, működik!
Nagyon gyorsan megérkezett a Tárhelyre!
